Havit.Blazor.Components.Web.Bootstrap.Documentation/Services/ComponentApiDocModelBuilder.cs

```diff
@@ -220,7 +222,7 @@ private bool ShouldIncludeMethod(MethodInfo methodInfo)
 		}
 
 		string name = methodInfo.Name;
-		if (name.StartsWith("set") || name.StartsWith("get") || ignoredMethods.Contains(name))
+		if (name.StartsWith("set") || name.StartsWith("get") || name.Contains("Clone") || ignoredMethods.Contains(name))
```

Comment:
Careful with the `name.Contains("Clone")`, still might be a valid member to include in documentation.
Did you try to filter-our all members with `CompilerGeneratedAttribute`?